bucket/termshark.json

```diff
@@ -0,0 +1,37 @@
+{
+    "version": "2.3.0",
+    "description": "A terminal UI for tshark, inspired by Wireshark",
+    "homepage": "https://github.com/gcla/termshark",
+    "license": "MIT",
+    "depends": "wireshark",
+    "notes": [
+        "Run 'scoop install wireshark' (from a elevated shell) to install 'tshark.exe' automatically"
+    ],
+
```

Comment:
```suggestion
    "depends": "extras/wireshark",
```

Reply:
@rashil2000 `tshark` is optional requirements, and need only for packet capture
also we have multiple `wireshark` package variants `wireshark-np` for example

Reply:
> @rashil2000 `tshark` is optional requirements, and need only for packet capture 

How can it be optional if it depends on it? It doesn't work without tshark. See: 

![image](https://user-images.githubusercontent.com/46838874/147586676-1f2b6575-02df-4269-a784-a307b44179d8.png)


> also we have multiple `wireshark` package variants `wireshark-np` for example

The extras bucket only has 'wireshark'